README.md

```diff
@@ -2,8 +2,15 @@
 
 ## Prerequisites
 
+### From scratch
```

Comment:
1. "jhbuild" section should be b/w "Prerequisites" and "From scratch"
2. Other distributions of linux and other OS like OSX also has its own package management system.
3. Unfortunately, too old version of jhbuild is install in Ubuntu 12.04 and emits following error. But not in Ubuntu 14.

> jhbuild build: unknown repository type system

So I'd like to suggest to edit like following;

```
### jhbuild (>= 3.5)
- If you want to get the latest version of jhbuild, refer to https://developer.gnome.org/jhbuild/stable/getting-started.html.en#getting-started-install (Note that section 2.1 is all you need and you don't need to run remains)
```